sample/firebase.json

```diff
@@ -4,10 +4,15 @@
       "target": "sample",
       "public": "dist/sample/dist/sample/browser",
       "ignore": [
-        "firebase.json",
-        "**/.*",
-        "**/node_modules/**"
+        "**/.*"
       ],
+      "headers": [{
+        "source": "*.[0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f].+(css|js)",
```

Comment:
Curious what this is?
Why can't we just use a quantifier here?

Reply:
Matching the sha'd file names from angular compiler, does the glob syntax in firebase.json support number of matches like regex? `[0-9a-f]{20}` I couldn't find it in the docs. Would be cleaner.